### PR TITLE
Deprecate NIRSpec workarounds

### PR DESCRIPTION
**Pull request by melanieclarke**
Adding deprecation notes for extraction recentering, S200A1/2 combination, and missing error workarounds.

### CODE REVIEW

File: NIRSPEC_General/nrs_missing_spectral_error_workaround.ipynb
Comment (sosey) [inline NIRSPEC_General/nrs_missing_spectral_error_workaround.ipynb:1]: @melanieclarke in this one, I had to change the name of the example file to get something that existed in MAST: 

> spec3_name = 'jw01128-o007_s00001_nirspec_f170lp-g235h-s200a2-subs200a2'

**Review by sosey — CHANGES_REQUESTED** [review]
Had some minor comments for 2 of the notebooks

**Comment by sosey** [comment]
@drlaw1558 has the JDOX table been updated for the deprecations here?

**Comment by drlaw1558** [comment]
I've noted these deprecations as such in the D2P table.  Going forward, the NIRSpec team should update the JDox known issues tables accordingly when notebooks are deprecated.

File: NIRSPEC_General/nrs_missing_spectral_error_workaround.ipynb
Comment (melanieclarke) [reply NIRSPEC_General/nrs_missing_spectral_error_workaround.ipynb:1]: Fixed.

File: NIRSPEC_Fixed_Slit/nrs_fss_s200a1_s200a2_workaround.ipynb
[the remark this answers is not in the document]
Comment (melanieclarke) [reply NIRSPEC_Fixed_Slit/nrs_fss_s200a1_s200a2_workaround.ipynb:1]: Thanks for pointing this out.  I added a note at the top about the pipeline version and context used at the time of writing, and modified the two cells at the bottom to work for either current or old pipeline naming system.  Let me know if you think it needs more clarification.

**Review by sosey — APPROVED** [review]

**Pull request merged**